spec/lib/fields/belongs_to_spec.rb

```diff
@@ -66,4 +66,19 @@
       end
     end
   end
+
+  describe "#resources" do
+    context "with `order` option" do
+      it "returns the resources in correct order" do
+        5.times { FactoryGirl.create(:customer) }
```

Comment:
We could use `create_list` here, like: `create_list(:customer, 5)`, which will create 5 customers.